crates/nu-command/tests/commands/mut_.rs

```diff
@@ -130,11 +130,19 @@ fn mut_path_upsert_list() {
 
 #[test]
 fn mut_path_operator_assign() {
-    let actual = nu!("mut a = {b:1}; $a.b += 3; $a.b -= 2; $a.b *= 10; $a.b /= 4; $a.b");
+    let actual =
+        nu!("mut a: record<b: number> = {b:1}; $a.b += 3; $a.b -= 2; $a.b *= 10; $a.b /= 4; $a.b");
 
     assert_eq!(actual.out, "5.0");
 }
 
+#[test]
+fn mut_path_operator_assign_should_error() {
+    let actual = nu!("mut a = {b:1}; $a.b += 3; $a.b -= 2; $a.b *= 10; $a.b /= 4; $a.b");
+
+    assert!(actual.err.contains("nu::shell::cant_convert"));
```

Comment:
This test doesn't actually check that every one of the `$a.b foo= bar` statements errors. Only one of them needs to error for the test to pass. I assume the intention is to make sure that all of them are not allowed

Reply:
the only one that's not allowed is the division since that would require changing the type from int to float